draft-ietf-rtcweb-jsep.xml

```diff
@@ -1568,6 +1584,28 @@ Suggested:
           remote candidate, connectivity checks will be sent to the new
           candidate.</t>
         </section>
+        <section anchor="sec.onicecandidate" numbered="true" toc="default">
+          <name>onicecandidate Event</name>
+          <t>The onicecandidate event is dispatched to the application when the
+          ICE agent has discovered a new allowed local ICE candidate during ICE
+          gathering, as outlined in
+          <xref target="sec.ice-gather-overview" format="default"></xref> and 
+          subject to the restrictions discussed in 
+          <xref target="sec.ice-candidate-policy" format="default"></xref>.
+          The new candidate is supplied as an IceCandidate object 
+          (<xref target="sec.ice-candidate-format" format="default"/>) in 
+          the event, and all of its fields <bcp14>MUST</bcp14> be filled in. 
+          This candidate will also be added to the current and/or pending local
+          description according to the rules defined for Trickle ICE.</t>
+          <t>The onicecandidate event will also be dispatched to indicate
+          end-of-candidates conditions, as defined in 
+          <xref target="RFC8838" format="default"/>. For these indications,
+          the IceCandidate's candidate field <bcp14>MUST</bcp14> be omitted.
+          The "m=" section index and MID fields may be set to signify whether
```

Comment:
Do you want to say "sdpMid and sdMLineIndex"?

Reply:
in JSEP we refer to these fields indirectly, see https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-24#section-3.5.2.1

Reply:
OK.